pandas/computation/__init__.py

```diff
@@ -0,0 +1,14 @@
+import warnings
+
+
+warnings.warn(
+    "The pandas.computation module is deprecated and will be removed in a future "
+    "version. Please import from the pandas.core.computation module instead.",
```

Comment:
So the "Please import from the pandas.core.computation module instead." should be removed IMO.